.travis.yml

```diff
@@ -2,5 +2,4 @@ language: clojure
 lein: lein2
 script: lein2 test-all
 jdk:
-  - openjdk7
-  - openjdk6
+  - oraclejdk8
```

Comment:
Nashorn is included as of Java 8. You wanted an "all included" cljs testing solution. Rhino depends on an external tool in doo.